templates/cart-booking-info.html

```diff
@@ -1,4 +1,7 @@
 <div class="eddbk-cart-item-booking-info">
-    <p><b>${from_label}</b> <time datetime="${start_datetime}">${start_text}</time></p>
-    <p><b>${until_label}</b> <time datetime="${end_datetime}">${end_text}</time></p>
+    <p>
+        <b>${from_label}</b> <time datetime="${start_datetime}">${start_text}</time>
+        <br style="display: block !important;"/>
```

Comment:
The [`<br />`][br] is an inline element, not block.

[br]: https://www.w3.org/TR/html5/textlevel-semantics.html#the-br-element

Reply:
Really? Then what makes it "break" the line?

Reply:
The fact that it represents a linebreak.

Reply:
Hah! No no, browsers apply a default style to it. One of those styles is what makes it break the line. Without those styles it's just an XML node.

@XedinUnknown So I did some research of my own and couldn't pinpoint what makes it break the line. But the problem is that some themes, for some reason, apply `display: none` to it, and applying `display: inline` seems to fix that. So I made the change anyway.

Reply:
What makes it break the line is the fact that it represents a linebreak, in the same way as the symbol "a" represents the letter "a" and that's why it gets rendered like that. That's all there is to it.